src/settings-web/src/icons/src/IconNames.ts

```diff
@@ -2,1756 +2,1857 @@
- ReleaseGateCheck = 'ReleaseGateCheck',
- ReleaseGateError = 'ReleaseGateError',
- M365InvoicingLogo = 'M365InvoicingLogo',
- RemoveFromShoppingList = 'RemoveFromShoppingList',
- ShieldAlert = 'ShieldAlert',
- FabricTextHighlightComposite = 'FabricTextHighlightComposite',
- Dataflows = 'Dataflows',
- GenericScanFilled = 'GenericScanFilled',
- DiagnosticDataBarTooltip = 'DiagnosticDataBarTooltip',
- SaveToMobile = 'SaveToMobile',
- Orientation2 = 'Orientation2',
- ScreenCast = 'ScreenCast',
- ShowGrid = 'ShowGrid',
- SnapToGrid = 'SnapToGrid',
- ContactList = 'ContactList',
- NewMail = 'NewMail',
- EyeShadow = 'EyeShadow',
- FabricFolderConfirm = 'FabricFolderConfirm',
- InformationBarriers = 'InformationBarriers',
- CommentActive = 'CommentActive',
- ColumnVerticalSectionEdit = 'ColumnVerticalSectionEdit',
- WavingHand = 'WavingHand',
- ShakeDevice = 'ShakeDevice',
- SmartGlassRemote = 'SmartGlassRemote',
- Rotate90Clockwise = 'Rotate90Clockwise',
- Rotate90CounterClockwise = 'Rotate90CounterClockwise',
- CampaignTemplate = 'CampaignTemplate',
- ChartTemplate = 'ChartTemplate',
- PageListFilter = 'PageListFilter',
- SecondaryNav = 'SecondaryNav',
- ColumnVerticalSection = 'ColumnVerticalSection',
- SkypeCircleSlash = 'SkypeCircleSlash',
- SkypeSlash = 'SkypeSlash',
- CustomizeToolbar = 'CustomizeToolbar',
- DuplicateRow = 'DuplicateRow',
- RemoveFromTrash = 'RemoveFromTrash',
- MailOptions = 'MailOptions',
- Childof = 'Childof',
- Footer = 'Footer',
- Header = 'Header',
- BarChartVerticalFill = 'BarChartVerticalFill',
- StackedColumnChart2Fill = 'StackedColumnChart2Fill',
- PlainText = 'PlainText',
- AccessibiltyChecker = 'AccessibiltyChecker',
- DatabaseSync = 'DatabaseSync',
- ReservationOrders = 'ReservationOrders',
- TabOneColumn = 'TabOneColumn',
- TabTwoColumn = 'TabTwoColumn',
- TabThreeColumn = 'TabThreeColumn',
- MicrosoftTranslatorLogoGreen = 'MicrosoftTranslatorLogoGreen',
- MicrosoftTranslatorLogoBlue = 'MicrosoftTranslatorLogoBlue',
- InternalInvestigation = 'InternalInvestigation',
- AddReaction = 'AddReaction',
- ContactHeart = 'ContactHeart',
- VisuallyImpaired = 'VisuallyImpaired',
- EventToDoLogo = 'EventToDoLogo',
- Variable2 = 'Variable2',
- ModelingView = 'ModelingView',
- DisconnectVirtualMachine = 'DisconnectVirtualMachine',
- ReportLock = 'ReportLock',
- Uneditable2 = 'Uneditable2',
- Uneditable2Mirrored = 'Uneditable2Mirrored',
- BarChartVerticalEdit = 'BarChartVerticalEdit',
- GlobalNavButtonActive = 'GlobalNavButtonActive',
- PollResults = 'PollResults',
- Rerun = 'Rerun',
- QandA = 'QandA',
- QandAMirror = 'QandAMirror',
- BookAnswers = 'BookAnswers',
- AlertSettings = 'AlertSettings',
- TableComputed = 'TableComputed',
- DecreaseIndentLegacy = 'DecreaseIndentLegacy',
- IncreaseIndentLegacy = 'IncreaseIndentLegacy',
- SizeLegacy = 'SizeLegacy'
-}
-
-// tslint:disable-next-line:deprecation
-export type IconNamesInput = keyof typeof IconNames;
\ No newline at end of file
+    PageLink = 'PageLink',
```

Comment:
Do you know what caused these to be reordered?  It makes diffing hard.

Reply:
https://github.com/microsoft/PowerToys/blob/master/doc/devdocs/settings-web.md